poppy/storage/cassandra/certificates.py

```diff
@@ -248,27 +248,41 @@ def get_certs_by_domain(self, domain_name, project_id=None,
             'domain_name': domain_name.lower(),
         }
 
-        query_string  = CQL_SEARCH_CERT_BY_DOMAIN
-
-        if project_id is not None:
-            query_string += "\nAND project_id = %(project_id)s "
-            args['project_id'] = project_id
-
-        if flavor_id is not None:
-            query_string += "\nAND flavor_id = %(flavor_id)s "
-            args['flavor_id'] = flavor_id
-
-        if cert_type is not None:
-            query_string += "\nAND cert_type = %(cert_type)s "
-            args['cert_type'] = cert_type
-
-        stmt = query.SimpleStatement(query_string,
+        stmt = query.SimpleStatement(CQL_SEARCH_CERT_BY_DOMAIN,
             consistency_level=self._driver.consistency_level)
         self.session.row_factory = dict_factory
         result = self.session.execute(stmt, args)
 
         try:
-            return ssl_certificate.SSLCertificate.init_from_dict(result[0])
+            cert_obj = result[0]
+            for k, v in cert_obj.items():
+                if k == "cert_details":
```

Comment:
This should be in a separate commit.

Reply:
If it is in a separate commit .. The new tests will fail? We can discuss.